namespaces-disallowlist.yaml

```diff
@@ -25,3 +25,10 @@
 - glean_dictionary
 - mach
 - "*_cirrus"
+- bedrock
+- firefox_desktop_background_tasks
+- firefox_desktop_background_defaultagent
+- hubs
+- relay_backend
+- gleanjs_docs
+- firefox_crashreporter
```

Comment:
Firefox crash reporter was only just [added](https://github.com/mozilla/probe-scraper/pull/803) in the last week.